### ISSUE TITLE
Fingerprint was invalid

### ISSUE THREAD

**Issue by holger-zeit** [figo-connect/python-figo#47]
My logs are spammed with constant 
```
Fingerprint "07:0F:14:AE:B9:4A:FB:3D:F8:00:E8:2B:69:A8:51:5C:EE:D2:F5:B1:BA:89:7B:EF:64:32:45:8F:61:CF:9E:33" was invalid
```
messages.

1. What should this tell me? Is it a real warning or can I ignore it?
2. If I can ignore it, how? If it's a real warning, what should be changed in the setup?

I'm on the latest 1.6.3 version.
Thanks!

**Comment by berend** [comment]
Thanks for reporting this!

We change recently the ssl certificate. To make the transition smoothly, we added both (then current now old fingerprint and the new upcoming now current fingerprint). You seeing the warning, that the old fingerprint does not match.

What this means:

1. You can ignore it, the sdk is wrong (Sorry!)
2. we will change the code to remove this outdated fingerprint and also change the warning to not show up when at least fingerprint is valid.

**Comment by holger-zeit** [comment]
Thanks for the fast response and for clearing things up!

**Comment by berend** [comment]
I messed up a bit though. I merged via web interface on github, then created and pushed the tags 1.6.4 later via cli. Result: a`python-figo 1.6.4.dev8` version and no `python-figo 1.6.4`.

A good reason to merge the next PR ...

**Comment by berend** [comment]
published `1.7.0` with proper naming and tagging
